services/core-api/app/cli_commands/export_permit_conditions.py

```diff
@@ -0,0 +1,103 @@
+import csv
+import datetime
+import os
+import uuid
+
+from app.api.mines.permits.permit_amendment.models.permit_amendment import (
+    PermitAmendment,
+)
+from app.api.mines.permits.permit_conditions.models.permit_conditions import (
+    PermitConditions,
+)
+from app.extensions import db
+
+
+def export_permit_conditions(permit_amendment_guid):
+    """
+    Export permit conditions for the specified permit amendment to a CSV file.
+    """
+    # Validate UUID format
+    try:
+        uuid.UUID(str(permit_amendment_guid))
+    except (ValueError, AttributeError, TypeError):
+        print(f'Invalid permit amendment GUID format: {permit_amendment_guid}')
+        return None
+
+    amendment = PermitAmendment.find_by_permit_amendment_guid(permit_amendment_guid)
+
+    if amendment is None:
+        print(f'Permit amendment with guid {permit_amendment_guid} not found.')
+        return
+
+    # Get all conditions in hierarchical order
+    conditions = PermitConditions.find_by_permit_amendment_id_ordered(amendment.permit_amendment_id)
+    
+    if not conditions:
+        print(f'No conditions found for permit amendment {permit_amendment_guid}')
+        return
+    
+    mine = amendment.mine
+    if not mine:
+        print(f'Permit amendment {permit_amendment_guid} has no associated mine')
+        return
+
+    permit = amendment.permit
+    if not permit:
+        print(f'Permit amendment {permit_amendment_guid} has no associated permit')
+        return
+
+    tasks = amendment.permit_extraction_tasks
+
+    latest_task = tasks[0] if tasks else None
+
+    document_name = ''
+    document_guid = ''
+
+    if latest_task and latest_task.permit_amendment_document:
+        doc = latest_task.permit_amendment_document
+        document_name = doc.document_name
+        document_guid = doc.document_manager_guid
+
+    # Create filename with timestamp
+    timestamp = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
+    filename = f'permit_conditions_{permit_amendment_guid}.csv'
```

Comment:
sonarcloud is flagging `timestamp` as unused- was it an accident to leave it out of the filename?